### ISSUE TITLE
Rdio audio ceases when app not "on top"

### ISSUE THREAD

**Issue by bryanwhite56** [chuot/rdio-scanner#398]
On my iPhone, it seems that the Rdio audio ceases when I go to reference another app, e.g. email, text
The app audio should continue even when other apps, including audio apps are in use.
I like to listen to music & when Rdio has something to say, sure, it can do so mixing with with the music.
ScannerRadio does this, not sure about Broadcastify apps? 
I prefer Rdio because of the minimal audio delay, the other apps delay 60-300 seconds.
Thanx

**Comment by renfrewcountyscanner** [comment]
See other posting for a response from another user.

**Comment by bryanwhite56** [comment]
Yes, same as 389.  This issue is 397 so this problem is not the same as
that one, it is that one. Thanx

I've never seen an app store upgrade on this app, hopefully soon
